blablacar-persistence/src/main/java/cz/fi/muni/pa165/teamred/entity/Place.java

```diff
@@ -0,0 +1,66 @@
+package cz.fi.muni.pa165.teamred.entity;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+/**
+ * @author Erik Horváth
+ */
+@Entity
+public class Place {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    @Column(nullable = false, unique = true)
+    private String name;
+
+//    private Set<Ride> rides;
+
+    public Place() {
+    }
+
+    public Place(Long id) {
+        this.id = id;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+//    public Set<Ride> getRides() {
+//        return Collections.unmodifiableSet(rides);
+//    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null) return false;
+        if (!(o instanceof Place)) return false;
+
+        Place other = (Place) o;
+
+        if (name == null) {
```

Comment:
Objects.equals(name,other.getName()) sa da pouzit namiesto name.equals nizsie a potom moze tento check ist prec. Objects.equals osetruje aj null argumenty: https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#equals(java.lang.Object,%20java.lang.Object)
Zjednodusuje to hlavne ked do equals davas viac fieldov.